File: templates/sort_file.js
function readArray(){
	const fs = require('fs');
	var file = fs.readFileSync('input.txt').toString('utf-8');
	let arr = file.split("\n");
	arr.pop();
	return arr;
}

arr = readArray();
var t0 = new Date();

// Sort here

var t1 = new Date() - t0;
console.log(arr);
console.log("Sorting took %d", t1);

File: templates/sort_file.py
import time


def readArray(path):
	route = []
	with open(path, 'r') as f:
		for line in f:
			route.extend(line.strip().split('>'))
	return route


if __name__ == "__main__":
	arr = readArray("input.txt")
	start = time.time()

	# Sort here

	end = time.time()
	print(arr)
	print("Time spent sorting: " + str(end-start))
